shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -142,9 +142,12 @@
       <li><button type="button" class="login-dev-account" data-name="Dave Dev User">
           Dave
       </button></li>
-      <li><button type="button" class="login-dev-account" data-name="Eve Dev User">
-          Eve
-      </button></li>
+      <li>
+        <div class="flex-box">
```

Comment:
This class name specifies implementation detail rather than semantics.

Also, do you actually need a `<div>` here, or could you make the parent `<li>` be the flexbox?

I would give it `class="custom-name"`, then not assign classes to the inner elements but instead nest their selectors inside `.custom-name`.